test/BlockManager.js

```diff
@@ -395,13 +417,16 @@ describe('BlockManager', function () {
       assert((await proposedBlock.valid) === false);
     });
 
-    it('no block should be confirmed in the previous epoch', async function () {
+    it('no block should be confirmed in the previous epoch and no peanalization for one block which is invalid', async function () {
```

Comment:
TestCase looks good. 
It does check our req. Can we rephrase this better maybe ? Like If No Block is valid in previous epoch, Stakers shouldnt be penalised. Its your call @GaurShekhar

Reply:
Ok will change the test case name.